Here is three commits to solve #5 issue

ramusus opened the pull request:
- made migration compatible with Django 1.7, added index on field_name column
- ability to change serialize name in settings: FIELD_HISTORY_SERIALIZER_NAME
- added json_nested_serializer for serializing fields from parent model

grantmcconnaughey commented on field_history/migrations/0001_initial.py:
```diff
@@ -12,7 +12,6 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('contenttypes', '0002_remove_content_type_name'),
```
Why was this removed?

ramusus replied:
it's for consistency with Django 1.7.5. At that time there was no such migration

ramusus replied:
and mention of this migration does not make sense here

grantmcconnaughey replied:
The `FieldHistory` model has a `ContentType` field, so we'll need to make sure that the migrations for `contenttypes` have completed before creating the `FieldHistory` table or else this migration will fail.

ramusus replied:
You are right, I will return it back

grantmcconnaughey commented:
Nice, lots of improvements here it looks like.

A few things I'd like to see before merging this:
* Tests. This took test coverage from 95% to 62%. Let's get that `json_nested_serializer.py` file completely tested.
* Documentation. Add a changelog entry for version 0.5.0 with the changes and put the `FIELD_HISTORY_SERIALIZER_NAME` setting in the documentation somewhere.

I need to look over it a bit more before giving more feedback, but those two things are a start.

ramusus commented:
Ok, agree will update PR

grantmcconnaughey commented:
Also, if you want to support Django 1.7.x then put that in `.travis.yml` so I can confirm all tests pass for it. [Support for Django 1.7 ended on December 1, 2015](https://www.djangoproject.com/download/), so I'm not going to do anything to add support for it. But, if you can get it working and it doesn't muddy up the codebase then I will accept the PR. Just know that if I have some feature I really want to add and it is Django 1.8+ only, then it will take precedence over keeping Django 1.7 compatibility.

ramusus commented:
@grantmcconnaughey I fixed coveralls and travis issues, added tests. I didn't update documentation, because this functionality should be supported by default, I suppose. 
Check, please is it ready to merge

grantmcconnaughey commented:
Awesome stuff, @ramusus. Give me the weekend to take a final look and write up some release notes.

grantmcconnaughey commented:
django-field-history 0.5.0 has been deployed. Thanks for the work!
